cmd/clusterctl/client/cluster/mover.go

```diff
@@ -360,9 +361,15 @@ func patchCluster(proxy Proxy, cluster *node, patch client.Patch) error {
 func (o *objectMover) ensureNamespaces(graph *objectGraph, toProxy Proxy) error {
 	ensureNamespaceBackoff := newWriteBackoff()
 	namespaces := sets.NewString()
-	for _, node := range graph.getNodesWithTenants() {
+
+	for _, node := range graph.getMoveNodes() {
 		namespace := node.identity.Namespace
 
+		// support cluster-wide objects
+		if namespace == "" {
+			continue
+		}
```

Comment:
We should add a test for this since it's new behavior.

Reply:
Added!